README.md

```diff
@@ -33,48 +48,132 @@ There is also a `prelude.rs` for helper code used by both (errors, traits, etc).
 * Deserialization for almost all supported types (see limitations section).
 * CDDL Generics - `foo<T> = [T]`, `bar = foo<uint>`
 * Length bounds - `foo = bytes .size (0..32)`
+* cbor in bytes - `foo_bytes = bytes .cbor foo`
```

Comment:
should we mention .default support as well?

Reply:
Yes. I might have forgotten some other features that we support too. There was quite a bit of work done since the last time it was updated.